src/rpc/rawtransaction.cpp

```diff
@@ -366,6 +367,100 @@ static RPCHelpMan getrawtransactionmulti() {
+            {
+                RPCResult{"for verbose = 0 if InstantSend Lock is known for specified txid",
+                     RPCResult::Type::STR, "data", "The serialized, hex-encoded data for 'txid'"
+                },
+                RPCResult{"for verbose = 1 if InstantSend Lock is known for specified txid",
+                    RPCResult::Type::OBJ, "", "",
+                    {
+                        {RPCResult::Type::STR_HEX, "txid", "The transaction id"},
+                        {RPCResult::Type::ARR, "inputs", "The inputs",
+                        {
+                            {RPCResult::Type::OBJ, "", "",
+                                {
+                                    {RPCResult::Type::STR_HEX, "txid", "The transaction id"},
+                                    {RPCResult::Type::NUM, "vout", "The output number"},
+                                },
+                            },
+                        }},
+                        {RPCResult::Type::STR_HEX, "cycleHash", "The Cycle Hash"},
+                        {RPCResult::Type::STR_HEX, "signature", "The InstantSend's BLS signature"},
+                    }},
+                RPCResult{"if no InstantSend Lock is known for specified txid",
+                     RPCResult::Type::STR, "data", "Just 'None' string"
+                },
+            }},
+        RPCExamples{
+            HelpExampleCli("getislocks", "'[\"txid\",...]'")
+            + HelpExampleRpc("getislocks", "'[\"txid\",...]'")
+        },
+    [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+    const NodeContext& node = EnsureAnyNodeContext(request.context);
+
+    UniValue result_arr(UniValue::VARR);
+    UniValue txids = request.params[0].get_array();
+    if (txids.size() > 100) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Up to 100 txids only");
+    }
+
+    int verbose = 1;
+    if (!request.params[1].isNull()) {
+        if (request.params[1].isBool()) {
+            verbose = request.params[1].get_bool() ? 1 : 0;
+        } else {
+            verbose = request.params[1].get_int();
+        }
+    }
+
+    const LLMQContext& llmq_ctx = EnsureLLMQContext(node);
+    for (const auto idx : irange::range(txids.size())) {
+        const uint256 txid(ParseHashV(txids[idx], "txid"));
+
+        if (const llmq::CInstantSendLockPtr islock = llmq_ctx.isman->GetInstantSendLockByTxid(txid); islock != nullptr) {
+            if (verbose == 1) {
+                UniValue objIS(UniValue::VOBJ);
+                objIS.pushKV("txid", islock->txid.ToString());
+                UniValue outputs(UniValue::VARR);
+                for (const auto out : islock->inputs) {
+                    UniValue outpoint(UniValue::VOBJ);
+                    outpoint.pushKV("txid", out.hash.ToString());
+                    outpoint.pushKV("vout", static_cast<int64_t>(out.n));
+                    outputs.push_back(outpoint);
+                }
```

Comment:
Missing push of array `inputs` to object (also consider renaming the variable to `inputs` to match the help text).

```suggestion
                }
                objIS.pushKV("inputs", outputs);
```

Reply:
good catch, see https://github.com/dashpay/dash/compare/9cbd1c63c14d1b9f22420fa45a64a8628fc5b00b..c9a82ba654af94a424fb09bebd33edf2ec9cdb7a